draft-thomson-quic-tls.md

```diff
@@ -724,6 +752,9 @@ server from being used in packet reflection attacks.
 
 A source address token is opaque and consumed only by the server.  Therefore it
 can be included in the TLS 1.3 pre-shared key identifier for 0-RTT handshakes.
+TODO: Could this be provided in TCP-based TLS 1.3 connections? If so that would
+mean that a TCP TLS 1.3 connection which was used to discover a server spoke
+QUIC could be used to enable a 0-RTT QUIC connection to that server.
```

Comment:
Oh, that's not a bad idea.  I think that it might work by accident in some implementations.  But others might prevent it simply by the fact that we are on a different 5-tuple.  There is no reason why it would not be permitted, and it might be a good idea.  We should explicitly permit it, otherwise we'll get complaints when some implementations do it and benefit.  We will also get complaints from the excessively paranoid, but there's nothing you can do about that :)